viadot/flows/genesys_to_adls.py

```diff
@@ -26,13 +31,57 @@ def add_timestamp(files_names: List = None, path: str = "", sep: str = "\t") ->
         df_updated.to_csv(os.path.join(path, file), index=False, sep=sep)
 
 
+@task(timeout=3600)
+def filter_userid(
+    files_names: list = None,
+    path: str = "",
+    sep: str = "\t",
+    userids: list = None,
+    apply_method: bool = False,
+) -> None:
+    """filter out the data frame by user ID.
+
+    Args:
+        files_names (list, optional): All file names of downloaded files. Defaults to "\t".
+        path (str, optional): Relative path to the file. Defaults to empty string.
+        userids (list, optional): List of all user IDs to select in the data frame. Defaults to None.
+        sep (str, optional): Separator in csv file. Defaults to None.
```

Comment:
Just a note - maybe change `sep` to `wite_sep` or modify the docstring to inform that this is the separator for the CSV file that will be created from this function.